pkg/operator/k8sutil/node.go

```diff
@@ -34,13 +34,9 @@ var validTopologyLabelKeys = []string{
 	"failure-domain.kubernetes.io",
 }
 
-// ValidNode returns true if the node (1) is schedulable, (2) meets Rook's placement terms, and
-// (3) is ready. False otherwise.
-func ValidNode(node v1.Node, placement rookalpha.Placement) (bool, error) {
-	if !GetNodeSchedulable(node) {
-		return false, nil
-	}
-
+// ValidNodeNoSched returns true if the node (1) meets Rook's placement terms,
+// and (2) is ready. False otherwise.
```

Comment:
I think a note here that a node *is* allowed to have a `NoSchedule` taint would be useful for the future

Reply:
fixed